R/endpoints.R

```diff
@@ -863,22 +924,29 @@ ecdc_ili <- function(regions,
   )
 }
 
-#' fetch FluSurv virological data
+#' Fetch FluSurv virological data
```

Comment:
issue: -> "FluSurv-Net hospitalization data".  "Virological" suggests results of virological testing.  (e.g., `fluview_clinical`, `fluview_public`)